vesc_driver/launch/vesc_driver_node.launch

```diff
@@ -4,17 +4,30 @@
   <arg name="node_name" default="vesc_driver_node" />
 
   <!-- Optionally launch in GDB, for debugging -->
-  <arg name="debug" default="false" />
-  <arg if="$(arg debug)" name="launch_prefix" value="xterm -e gdb --args" />
-  <arg unless="$(arg debug)" name="launch_prefix" value="" />
+  <!-- <arg name="debug" default="false" /> -->
+  <!-- <arg if="$(arg debug)" name="launch_prefix" value="xterm -e gdb args" /> -->
+  <!-- <arg unless="$(arg debug)" name="launch_prefix" value="" /> -->
```

Comment:
Remove unused code. `git` will keep the history and commenting out unused sections just leads to confusion.